### ISSUE TITLE
PhpDoc array key completion not working

### ISSUE THREAD

**Issue by King2500** [klesun/deep-assoc-completion#23]
![image](https://user-images.githubusercontent.com/4896363/34101166-189cb538-e3e5-11e7-9dfe-6bd961c81cf6.png)

```
PhpStorm 2017.3.1
Build #PS-173.3942.32, built on December 12, 2017
JRE: 1.8.0_152-release-1024-b8 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
```

**Comment by klesun** [comment]
Thanks for your report. I'll look into it as soon as possible.

**Comment by klesun** [comment]
Fix: a76d0eff1593e40e679b6814b35b60a0b9701a09
https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/41602

It was caused by recent change in phpstorm PSI structure: topmost level code block (namespace) was changed from `GroupStatementImpl` to `GroupStatementSimpleImpl` somewhere between `2017.1.4` and `2017.3.1`.
